free-3.md

```diff
@@ -10,41 +10,91 @@ _What need is met by your idea? Who are you helping? What is that person's
 
+I used to enjoy listening to random noise in order to fall asleep.
+Now that I know more math and cs, I thought about writing a program to generate
+random noise.
+I've also been having trouble sleeping because I've been sick, so I thought
+about random noise again.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Generate random noise for relaxation.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The interface should be graphical. Users can draw the top of a frequency 
+spectrum like those displayed 
+[here](https://en.wikipedia.org/wiki/Colors_of_noise).
+
+Users can also utilize a slider to change left-right balance for stereo volume.
+
+The graphical interface is simple to use and hopefully intuitive.
+People who are interested in random noise also use colors to name noise,
+so a graphical interface may be helpful for generating more complex noise.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program would use the currently displayed frequency spectrum to generate
+noise.
+The noise should stay updated as the user modifies the spectrum.
+So the noise should change with the drawing without interruptions.
+
+If an error occurs, the noise should stop as really terrible sounds could
+result from the error. Then an error message should appear on the graphical
+interface.
+
+A possible error could be a strange spectrum that is discontinuous.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to generate white, pink, brown, blue, and violet noise 
+since the spectra are just straight lines.
+But it should be easy to make a make a preferred noise since the user gets
+continuous feedback.
+
+It should be possible to create noise of discrete frequencies, although it
+should be difficult.
+
+It would be almost impossible to make music.
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are applications that play preset colors of noise.
+There are also websites like [this one](https://www.random.org/audio-noise/)
+that generates random noise with some customization.
+Along a similar domain, there are many websites that overlay ambient noises
+like [this one](http://soundrown.com/).
+I have not found an app or website that lets you customize noise based on
+frequency spectrum, which may be too niche.
 
```

Comment:
I think the niche domain you have found would definitely be accepting of a dsl to blossom. The apps you list are cool/useful but the spectrum interaction you bring up, while possibly difficult, is uniquely cool/useful.